modules/service/src/main/scala/lucuma/odb/graphql/mapping/MutationMapping.scala

```diff
@@ -252,15 +251,6 @@ trait MutationMapping[F[_]] extends Predicates[F] {
       child
     )
 
-  def proposalAttachmentResultSubquery(pid: Program.Id, aTypes: List[Tag], limit: Option[NonNegInt], child: Query) =
-    mutationResultSubquery(
-      predicate = And(Predicates.proposalAttachment.program.id.eql(pid), Predicates.proposalAttachment.attachmentType.in(aTypes)),
-      order = OrderSelection[Tag](ProposalAttachmentType / "attachmentType"),
-      limit = limit,
-      collectionField = "proposalAttachments",
-      child
-    )
```

Comment:
This was unused and I suspected it was meant to be deleted some time ago, so I removed it.